rotkehlchen/globaldb/upgrades/v6_v7.py

```diff
@@ -1,47 +1,74 @@
 import logging
 from pathlib import Path
 from typing import TYPE_CHECKING
-from rotkehlchen.errors.misc import DBUpgradeError
-
 
+from rotkehlchen.errors.misc import DBUpgradeError
 from rotkehlchen.logging import RotkehlchenLogsAdapter
 
 if TYPE_CHECKING:
+    from collections.abc import Callable
     from rotkehlchen.db.drivers.gevent import DBConnection, DBCursor
 
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
 
+def enter_exit_debug_log(upgrade_function: 'Callable') -> 'Callable':
+    """Decorator to debug log enter and exit events of a DB upgrade function"""
+    def log_wrapped(cursor: 'DBCursor') -> None:
+        log.debug(f'Enter {upgrade_function.__name__}')
+        upgrade_function(cursor)
+        log.debug(f'Exit {upgrade_function.__name__}')
+    return log_wrapped
```

Comment:
can't this be used also in the user db upgrades? I would move it to an utils files and use it in other places too

Reply:
I will move this decorator to other files also in a different PR because it will affect many files.